### DISCUSSION TITLE
Uploading many Feature Collections

### DISCUSSION THREAD

**Discussion by S-Bronson** [samapriya/awesome-gee-community-datasets#217] [Q&A]
Hello all,

I am wondering if I can have some help/suggestions for dealing with large datasets of Feature Collections (coming from 388 shapefiles totaling over 100 GB in size). Each shp has the same set of fields and can be merged together.
The raw data can be found here,
https://opendata.nfis.org/downloads/forest_change/CA_Forest_Satellite_Based_Inventory_2020.zip

Once uploaded to Awesome-GEE, I can imagine that users of the data would prefer everything to be in one Feature Collection for ease of analysis. I have uploaded each shp to GEE and flattened all into one Feature Collection, but uploading that single combined Feature Collection has taken more than 40 hours and been attempted twice.

I am hoping someone can assist in this upload as i want the data to be as user-friendly as possible.

Cheers,
   Spencer

**Comment by samapriya** [comment]
Sorry I never got back to you on this but this is now part of the community collection. Thank you for contributing
